picard/plugin.py

```diff
@@ -39,6 +39,7 @@
 from picard.const import USER_PLUGIN_DIR, PLUGINS_API
 from picard.util import load_json
 
+PLUGIN_ACTION_UPDATE = 1
```

Comment:
Since you need this here and in `picard/ui/options/plugins.py`, i would set those in `picard/const.py`:

```python
PLUGIN_ACTION_NONE, PLUGIN_ACTION_UPDATE, PLUGIN_ACTION_INSTALL = range(3)
```

and import those where needed:

```
from picard.const import USER_PLUGIN_DIR, PLUGINS_API, PLUGIN_ACTION_UPDATE
```



Note: we could start to use Python3 Enum for this, but i'm not sure which version supports what etc..